recipes/obake/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "0.1" %}
+
+package:
+  name: obake
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluescarni/obake/archive/v{{ version }}.tar.gz
+  sha256: 946de817940421d8af8c614c289b74dc6e221ea803fc2b3dacc30e63d7dc3246
+
+build:
+  number: 0
+  skip: true   # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang      # [win]
+    - ninja      # [win]
+
+outputs:
+  - name: obake
+    build:
+      run_exports:
+        # SO name changes at minor rev bumps.
+        - {{ pin_subpackage('obake', max_pin='x.x') }}
```

Comment:
This should go in `obake-devel`

Reply:
But in the openblas example the ``run_exports`` is not in the ``devel`` package, is it?

Reply:
True. That should be fixed in the openblas recipe.